covid/tests.py

```diff
@@ -0,0 +1,47 @@
+import numpy
+import pandas
+import pytest
+
+import arviz
+
+import covid.data
+import covid.models.generative
+
+
+class TestDataUS:
+    def test_get_raw(self):
+        df_raw = covid.data.get_raw_covidtracking_data()
+        assert isinstance(df_raw, pandas.DataFrame)
+
+    def test_process(self):
+        df_raw = covid.data.get_raw_covidtracking_data()
+        df_processed = covid.data.process_covidtracking_data(df_raw, pandas.Timestamp('2020-06-25'))
+        assert isinstance(df_processed, pandas.DataFrame)
+
+
+class TestGenerative:
+    def test_build(self):
+        df_raw = covid.data.get_raw_covidtracking_data()
+        df_processed = covid.data.process_covidtracking_data(df_raw, pandas.Timestamp('2020-06-25'))
+        model = covid.models.generative.GenerativeModel(
+            region='NY',
+            observed=df_processed.xs('NY')
+        )
+        model.build()
```

Comment:
Should this one have an assertion? If were just checking if the model would build then would suggest adding a comment making that intent clear

Reply:
Done!
I added TODOs. Increasing test coverage should be a follow-up PR.
This is just to get a CI up in the first place.